### DISCUSSION TITLE
Add an option to sort servers on community website

### DISCUSSION THREAD

**Discussion by goosenation** [gameserverapp/Platform#1020] [Suggestions]
Currently, the server order in the game server slider block is by their ID.

Would be nice if there were an option to sort them differently.

One thing that comes to mind is by the custom server order in the cluster list on the dashboard.

**Comment by Syroxxa** [comment]
I agree this would be awesome :)

**Comment by goosenation** [comment]
I just found another possible option for sorting the servers:

[Dashboard] Servers -> Integration Mod - Vote Screen
![image](https://github.com/user-attachments/assets/6a513fce-4a49-43d2-b048-59412fa28de5)
